lnbits/.env

```diff
@@ -0,0 +1,80 @@
+QUART_APP=lnbits.app:create_app()
+QUART_ENV=development
+QUART_DEBUG=true
+
+HOST=127.0.0.1
+PORT=5000
+
+LNBITS_ALLOWED_USERS=""
+LNBITS_ADMIN_USERS=""
+# Extensions only admin can access
+LNBITS_ADMIN_EXTENSIONS="ngrok"
+LNBITS_DEFAULT_WALLET_NAME="LNbits wallet"
+
+LNBITS_AD_SPACE="" # csv ad image filepaths or urls, extensions can choose to honor
+LNBITS_HIDE_API=false # Hides wallet api, extensions can choose to honor
+
+# Disable extensions for all users, use "all" to disable all extensions
+LNBITS_DISABLED_EXTENSIONS="amilk"
+
+# Database: to use SQLite, specify LNBITS_DATA_FOLDER
+#           to use PostgreSQL, specify LNBITS_DATABASE_URL=postgres://...
+#           to use CockroachDB, specify LNBITS_DATABASE_URL=cockroachdb://...
+# for both PostgreSQL and CockroachDB, you'll need to install
+#   psycopg2 as an additional dependency
+LNBITS_DATA_FOLDER="./data"
+LNBITS_DATABASE_URL="postgres://postgres:myPassword@localhost:5432/lnbits"
```

Comment:
you will want to switch this to: `LNBITS_DATABASE_URL="postgres://postgres:myPassword@playground-postgres:5432/lnbits"`

Reply:
ok. 

Looks like we need to put back the virtualenv. the 3.10 library conflicts which are built into the ubuntu OS are giving massive issues with the 3.8 install and the only way to properly override this is with a virtual env. there isn't a clean way to do this by installing a fresh copy of 3.8, it won't override the 3.10 . 

So I guess back to the drawing board... what a waste of the last 20 hrs... oh well.

Reply:
could you not just switch to an image based on `python:3.8.11-bullseye` ? Then you can have the version of python you want and  debian based for any other things you know how to install.

Reply:
I did try your suggestion for `python:3.8.11-bullseye` but that didn't work, i think it was because ubuntu:latest had too new of a version so, I switched ubuntu:20.04, which has a stable release of python3.8 and is what we were running on last year. My bad, I thought latest was 20.04, turns out its actually 22.

20.04 - that seems to load - see latest commit. now its just a matter of fixing the postgres link and then i think we're done after some clean up.